src/cards/ha-media_player-card.html

```diff
@@ -248,8 +248,20 @@
 
   playerObjChanged: function (playerObj) {
     var picture = playerObj.stateObj.attributes.entity_picture;
+    var dummy;
     if (picture) {
-      this.$.cover.style.backgroundImage = 'url(' + picture + ')';
+      dummy = document.createElement('IMG');
```

Comment:
We wrap this in an `if (this.$.cover.style.backgroundImage != 'url(' + picture + ')'), or something along those lines, so that once the image loads successfully, it doesn't keep reloading it.